### ISSUE TITLE
Centaur Warrunner's Hoof Stomp doesn't trigger Arcane Curse silence

### ISSUE THREAD

**Issue by Undead94100** [ValveSoftware/Dota2-Gameplay#19072]
### Ability name

Arcane Curse & Hoof Stomp

### Description

When Centaur Warrunner uses Hoof Stomp while afflicted by Silencer's Arcane Curse, the Curse's 2 second silence upon spell cast doesn't trigger, whereas both Double Edge and Stampede do, as expected.

### Example Match ID (and possibly Timestamp)

7774886987 (2:10)

**Comment by Dum0sss** [comment]
it casts on last tick of animation, so he is probably casting it and then procing debuff, is that right?

**Comment by Undead94100** [comment]
No, I do mean it actually does nothing. Centaur can immediately cast spells after Hoof Stomp, even though he should be silenced.

https://github.com/ValveSoftware/Dota2-Gameplay/assets/112407737/5fd28034-2f94-4561-8acd-467276e4cb94

**Comment by Dum0sss** [comment]
then its a bug, coming from his hoof coding.
